Dockerfile.builder

```diff
@@ -10,28 +12,27 @@ LABEL maintainer="Plone Community <dev@plone.org>" \
 RUN <<EOT
     set -e
     apt update
-    apt install -y --no-install-recommends python3 build-essential git ca-certificates
-    npm install --no-audit --no-fund -g yo @plone/generator-volto@alpha
-    mkdir /app
-    chown -R node:node /app
+    apt install -y --no-install-recommends python3 python3-pip build-essential git ca-certificates pipx
+    npm install --no-audit --no-fund -g mrs-developer
     rm -rf /var/lib/apt/lists/*
+    pipx run cookiecutter gh:plone/cookiecutter-volto addon_name=app --no-input
```

Comment:
@sneridagh @ericof Will we update this to use cookieplone and the volto template in cookiecutter-plone?

Reply:
@ericof do we have the internal variable for cookieplone to not generate the addon?